apps/dcos_l4lb/src/dcos_l4lb_mgr.erl

```diff
@@ -460,9 +470,21 @@ healthy_vips(VIPs, Nodes, Tree)
     VIPs;
 healthy_vips(VIPs, Nodes, Tree) ->
     Agents = agents(VIPs, Nodes, Tree),
-    lists:map(fun ({VIP, BEs}) ->
-        {VIP, healthy_backends(BEs, Agents)}
-    end, VIPs).
+    VIPs1 = lists:map(fun ({VIP, BEs}) ->
```

Comment:
nit: I know that `BEs` was here before. Could you please use `Backends` instead of `BEs` in variable names here?

Reply:
I welcome the `Backends` change but would also like to know what @urbanserj  thinks.

Reply:
`BE`/`BEs` is a widely-used variable name across `dcos_l4lb` application. I'm fine with `Backends`, but I think in that case you will have to change it in all functions in all modules.

Reply:
That makes me feel like not doing it, so we can focus on the MUST do for getting this PR merged.